normandy/recipes/tests/test_models.py

```diff
@@ -172,12 +173,30 @@ def test_update_signature(self, mocker):
         ]
 
         recipe = RecipeFactory(signed=False)
-        assert recipe.signature is None
```

Comment:
Not when we have a working (or mocked) Autograph, since we automatically sign in `save()` now.